src/configure/extract_from_c.ml

```diff
@@ -56,10 +57,23 @@ let headers = "\
 #define __USE_MINGW_ANSI_STDIO 1
 #include <stdio.h> /* see: https://sourceforge.net/p/mingw-w64/bugs/627/ */
 #endif
+#ifdef _MSC_VER
+#include <complex.h> /* see https://docs.microsoft.com/en-us/cpp/c-runtime-library/complex-math-support?view=msvc-160 */
+#endif
 #include <stdint.h>
 #include <stdbool.h>
 #include <inttypes.h>
 #include <caml/mlvalues.h>
+
+#ifdef _MSC_VER
+typedef _Lcomplex            longdoublecomplex_t;
```

Comment:
type names like `longdoublecomplex_t` are horrible to read. Use `_` and prefix them with `ctypes_`, just follow the usual convention, see e.g: https://github.com/ocamllabs/ocaml-ctypes/blob/261fe071fad17ab323d8d2b82df2aec593e64e3f/src/ctypes/ctypes_complex_compatibility.h#L20 )

Reply:
Thanks. Will change.